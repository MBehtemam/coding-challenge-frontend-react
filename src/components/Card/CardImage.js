import styled from 'styled-components';

const CardImage = styled.img`
  border-top-left-radius: 10px;
  border-top-right-radius: 10px;
`;
export default CardImage;
