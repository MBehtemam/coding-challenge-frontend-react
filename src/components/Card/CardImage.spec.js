import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import CardImage from './CardImage';

describe('Test suits for Card Image', () => {
  it('should match with snapshot', () => {
    const tree = renderer.create(<CardImage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
