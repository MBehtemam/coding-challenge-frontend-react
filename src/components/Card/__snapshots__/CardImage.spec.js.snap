// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Card Image should match with snapshot 1`] = `
.c0 {
  border-top-left-radius: 10px;
  border-top-right-radius: 10px;
}

<img
  className="c0"
/>
`;
